src/main/java/apoc/convert/Convert.java

```diff
@@ -163,5 +165,76 @@ else if (list.getClass().isArray() && !list.getClass().getComponentType().isPrim
 	public List<Relationship> toRelationshipList(@Name("list") Object list) {
         return convertToList(list, Relationship.class);
 	}
-	
+
+    @UserFunction
+    @Description("apoc.convert.toInteger(value) | tries it's best to convert the value to an integer")
+    public Long toInteger(@Name("object") Object obj) {
+        if (obj == null || obj.equals("")) {
+            return null;
+        }
+
+        Types varType = Types.of(obj);
```

Comment:
you can also add teh try-catch here, and return null in case of exception

Reply:
This seems pointless - Types.of() implementation returns UNKNOWN type as a fallback. It does not throw any exceptions AFAIK.